observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc

```diff
@@ -24,7 +24,7 @@ include::modules/distr-tracing-features.adoc[leveloffset=+1]
 include::modules/distr-tracing-architecture.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources_distributed-tracing-architecture"]
+[id="additional-resources_distributed-tracing-architecture_{context}"]
```

Comment:
In this case just adding the context variable without removing the manually typed one will create an ID where the context is spelled out twice:
```suggestion
[id="additional-resources_{context}"]
```

Reply:
Thank you, Sebastian.

:slightly_smiling_face: I think what is happening is when you mention issues that require global changes, I go through all of these docs and fix these issues in many other files, at which point you review those additional files and point out other issues.

I have fixed all of these as you requested.

I plan to review these docs for modular docs during a later month to prepare the docs for migration to DITA. So I have been happy to fix some of the issues in this PR.